skorecard/reporting/plotting.py

```diff
@@ -255,3 +255,64 @@ def plot_bucket(self, column, line="event_rate", format=None, scale=None, width=
             width=width,
             height=height,
         )
+
+
+def weight_plot(stats: pd.DataFrame):
+
+    """
+    Generates a weight plot(plotly chart) from `stats`
+    Example:
+
+    ```from skorecard.datasets import load_uci_credit_card
+    from skorecard.bucketers import EqualFrequencyBucketer
+    from skorecard.linear_model import LogisticRegression
+
+    from sklearn.pipeline import Pipeline
+    from sklearn.preprocessing import OneHotEncoder
+    X, y = load_uci_credit_card(return_X_y=True)
+    pipeline = Pipeline([
+        ('bucketer', EqualFrequencyBucketer(n_bins=10)),
+        ('clf', LogisticRegression(calculate_stats=True))
+    ])
+    pipeline.fit(X, y)
+    assert pipeline.named_steps['clf'].p_val_coef_[0][0] > 0
+    stats = pipeline.named_steps['clf'].get_stats()
+
+    weight_plot(stats)```
+    """
+    fig = go.Figure()
+
+    stats["[0.025"] = stats["Coef."] - 1.96 * stats["Std.Err"]
```

Comment:
To be honest, I'm not a fan of these column names. Maybe something like `conf_interval_0.025` etc.